web/public/scripts/ua_latest.js

```diff
@@ -0,0 +1,152 @@
+!function() {
+  "use strict";
+  (t => {
+    const {
+      screen: { width: e, height: a },
+      navigator: { language: r },
+      location: n,
+      localStorage: i,
+      document: c,
+      history: o
+    } = t;
+    const { hostname: s, href: u } = n;
+    const { currentScript: l, referrer: d } = c;
+
+    if (!l) return;
+
+    const m = "data-",
+      f = l.getAttribute.bind(l),
+      h = f(m + "website-id"),
+      p = f(m + "host-url"),
+      g = f(m + "tag"),
+      y = "false" !== f(m + "auto-track"),
+      b = "true" === f(m + "exclude-search"),
+      v = f(m + "domains") || "",
+      w = v.split(",").map(t => t.trim()),
+      S = `${(p || "https://api-gateway.umami.dev" || l.src.split("/").slice(0, -1).join("/")).replace(/\/$/, "")}/api/send`,
+      N = `${e}x${a}`,
+      T = /data-umami-event-([\w-_]+)/,
+      A = m + "umami-event",
+      x = 300;
+
+    const O = t => {
```

Comment:
do we have documentation on what these methods do?

Reply:
no this is their tracking script, but self-hosted to avoid content blockers

Reply:
do you get this script from somewhere? can we attach the link to this PR for reference, thanks!

Reply:
https://cloud.umami.is/script.js